src/docfx/config/ops/OpsMetadataRuleConverter.cs

```diff
@@ -116,16 +109,38 @@ public static string GenerateJsonSchema(string rulesContent, string allowlistsCo
 
                 if (rulesInfo.TryGetValue("List", out var listRuleInfo) &&
                     listRuleInfo != null &&
-                    TryGetTaxonomy(attribute, listRuleInfo.List, taxonomies, out var taxonomy))
+                    TryGetTaxonomy(attribute, listRuleInfo.List, taxonomies, out var taxonomy, out var enumValues))
                 {
-                    schema.enumDependencies.Add($"{attribute}[0]", taxonomy);
-
-                    if (rulesInfo.TryGetValue("Match", out var matchRuleInfo) &&
-                        !string.IsNullOrEmpty(matchRuleInfo.Value) &&
-                        !schema.enumDependencies[$"{attribute}[0]"].ContainsKey(matchRuleInfo.Value))
+                    // if just plain enum, extend property
+                    if (enumValues != null && enumValues.Length > 0)
                     {
-                        schema.enumDependencies[$"{attribute}[0]"].Add(matchRuleInfo.Value, null);
+                        var enumType = new
```

Comment:
do we need consider whether it's array or string? directly adding `iterms` seems risky

Reply:
ok